packages/database/src/migrations/20210824045814-AddPacMossiRawDataDownload-modifies-data.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+import {
+  codeToId,
+  arrayToDbString,
+  insertObject,
+  generateId,
+  findSingleRecord,
+  deleteObject,
+} from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const PROJECT_NAME = 'PacMOSSI';
+const COUNTRY_CODES = ['FJ', 'KI', 'NR', 'NU', 'PG', 'SB', 'TV', 'TO', 'VU'];
+const DASHBOARD_NAME = 'PacMOSSI Raw Data Downloads';
+const DASHBOARD_ITEM_NAME = 'Download Vector Surveillance Survey Data';
+
+const getCountryDashboardCode = countryCode =>
+  `${countryCode}_${PROJECT_NAME}_${DASHBOARD_NAME.split(' ').join('_')}`;
```

Comment:
```suggestion
  `${countryCode}_${DASHBOARD_NAME.split(' ').join('_')}`;
```

Seems like unnecessary to have two `pacmossi` in code `FJ_PacMOSSI_PacMOSSI_Raw_Data_Downloads`,

Reply:
Fixed!